src/TMap.cpp

```diff
@@ -2388,6 +2433,7 @@ void TMap::slot_replyFinished(QNetworkReply* reply)
                     if (!pHost) {
                         qWarning() << "TMap::slot_replyFinished( QNetworkReply * ) ERROR - NULL Host pointer - something is really wrong!";
                         mXmlImportMutex.unlock();
```

Comment:
This `unlock` call is already part of the `cleanup` lambda if I read the code correctly.

Reply:
Fixed.